js/test/strings-ext/testlistfmt_pa.js

```diff
@@ -145,7 +143,7 @@ module.exports.testlistfmt_pa = {
     
         test.expect(2);
         test.ok(fmt !== null);
-        test.equal(fmt.format(["واحد", "اثنان"]), "واحد, اثنان");
+        test.equal(fmt.format(["واحد", "اثنان"]), "واحد ਅਤੇ اثنان");
```

Comment:
This is not right. The list items are in Arabic script, but the translation of "and" (ਅਤੇ) is written in the Devanagari script. I think the pa language normally uses Devanagari, but the the pa-Arab-PK locale uses Arabic. In this case, the file js/data/locale/pa/Arab/list.json is not correct.

Reply:
The Extract tool code should generate `pa/Arab/list.json` file. but It wasn't.  It works with cldr 34 and the data haven't changed much in cldr 36. It's weird. I need to look at the tool code more.